specification/README.md

```diff
@@ -0,0 +1,13 @@
+# Ably Asset Tracking Specification
+
+## Overview
+
+This document is meant to be a formal, canonical specification describing the Ably Asset Tracking solution.
```

Comment:
In line with the wider description in #45, which we've discussed outside scope of this PR but have agreed this PR represents the starting point for, I think we can refine this paragraph and make it a little bit more assertive:

```suggestion
This document is the formal, canonical specification describing the ecosystem, including requirements, shared by components of Ably's [Asset Tracking solution](https://ably.com/solutions/asset-tracking) (AAT).
```

Reply:
https://github.com/ably/ably-asset-tracking-common/pull/56/commits/77941d29ac2675a30b08a4e76157d9d38349005b